plugins/catalog-backend/src/ingestion/types.ts

```diff
@@ -102,7 +102,7 @@ export type DescriptorEnvelope = {
   /**
    * Optional metadata related to the entity.
```

Comment:
Comment still says it is optional

Reply:
This change isn't going to happen. Eventually types and validators are going to migrate to core package. For know I'm going to just copy types into front-end part.